src/anki/scene/components/MoveComponent.h

```diff
@@ -146,13 +146,21 @@ class MoveComponent : public SceneComponent
 		ANKI_ASSERT(point.w() == 0.0f);
 		const Vec4 j = Vec4(0.0f, 1.0f, 0.0f, 0.0f);
 		const Vec4 vdir = (point - m_ltrf.getOrigin()).getNormalized();
-		const Vec4 vup = j - vdir * j.dot(vdir);
+		const Vec4 vup = (j - vdir * j.dot(vdir)).getNormalized();
 		const Vec4 vside = vdir.cross(vup);
 
 		Mat3x4& rot = m_ltrf.getRotation();
 		rot.setColumns(vside.xyz(), vup.xyz(), (-vdir).xyz());
 		markForUpdate();
 	}
+
+	void setPitchYawRoll(F32 radPitch, F32 radYaw, F32 radRoll) 
+	{
+		const Vec4 forward = Vec4(	cos(radYaw) * cos(radPitch),
```

Comment:
hmmm. That's very weird it appears it's not aligned. If you run the format_source.sh script and it didn't work maybe it's something with the line endings on Windows. In the 0edd903e5125c578b963f0879233f71d1aa9a6eb commit I've added a .gitattributes file that tells git to treat line endings as Unix line endings. Not sure if that will help

Reply:
I'll give it a go. If it doesn't work I'll just initialize the whole thing in one line for the moment.